packages/plugins/plugin-navtree/src/components/L1Panels.tsx

```diff
@@ -166,31 +165,27 @@ const L1PanelCollection = ({ item, path, currentItemId }: L1PanelProps) => {
       {collectionItems
         .filter((item) => l0ItemType(item) === 'tab')
         .map((item) => (
-          <L1Panel key={item.id} item={item} path={groupPath} currentItemId={currentItemId} />
+          <L1Panel key={item.id} item={item} path={groupPath} {...props} />
         ))}
     </>
   );
 };
 
-export const L1Panels = ({
-  topLevelItems,
-  path,
-  currentItemId,
-  onBack,
-}: Pick<L1PanelProps, 'onBack'> & {
+export type L1PanelsProps = Pick<L1PanelProps, 'open' | 'currentItemId' | 'onBack'> & {
   topLevelItems: Node<any>[];
   path: string[];
-  currentItemId: string;
-}) => {
+};
+
+export const L1Panels = ({ topLevelItems, onBack, ...props }: L1PanelsProps) => {
   return (
     <>
       {topLevelItems.map((item) => {
         const type = l0ItemType(item);
         switch (type) {
           case 'collection':
-            return <L1PanelCollection key={item.id} item={item} path={path} currentItemId={currentItemId} />;
+            return <L1PanelCollection key={item.id} item={item} {...props} />;
           case 'tab':
-            return <L1Panel key={item.id} item={item} path={path} currentItemId={currentItemId} onBack={onBack} />;
+            return <L1Panel key={item.id} item={item} onBack={onBack} {...props} />;
```

Comment:
prefer the rightmost term of an event handler is named after a native interaction event; here it should be `onBackClick`